pages/success-banner/index.js

```diff
@@ -22,11 +36,13 @@ const Banner = () => (
       </h1>
     </div>
     <div className="horizontal center">
-      <Button href="https://github.com/resin-io/etcher">
+      <Button href="https://github.com/resin-io/etcher"
+        event="click Github star button">
         <img className="icon github" src="/static/social/octocat.png" />
         Star on Github
       </Button>
-      <Button href="https://twitter.com/intent/tweet?text=%23etcher">
+      <Button href="https://twitter.com/intent/tweet?text=%23etcher"
+        event="click Tweet button">
```

Comment:
I wonder if it'd make sense to move the "click " and " button" text into the Button's on-click handler?
So that `event="click Tweet button"` here could be changed to `label="Tweet"` and line 24 would change to `onClick={ eventLog('click '+this.props.label+' button') }>`